index.js

```diff
@@ -1,7 +1,18 @@
-const emailLog = require('./email_log.json');
+const emailLog = require("./email_log.json");
 
 function countUniqueEmails(emailFile) {
   // do work here
+
```

Comment:
Please use meaningful naming conventions, `p` and `c` should be named based on the value that they represent.